README.md

```diff
@@ -15,7 +15,9 @@ C++ runtime for [Rive](https://rive.app). Provides these runtime features:
 We use [premake5](https://premake.github.io/). The Rive dev team primarily works on MacOS. There is some work done by the community to also support Windows and Linux. PRs welcomed for specific platforms you with to support! We encourage you to use premake as it's highly extensible and configurable for a variety of platforms.
 
 ## Build
-In the ```rive``` directory, run ```build.sh``` to debug build and ```build.sh release``` for a release build.
+In the ```rive-cpp``` directory, run ```build.sh``` to debug build and ```build.sh release``` for a release build.
+
+If you've put the `premake5` executable in the `rive-cpp/build` folder, you can run it with `PATH=.:$PATH ./build.sh`
```

Comment:
Although not directly related to the fix in this PR, I think it helps users that don't want to put alpha software in their main PATH.